src/suspendplugin.c

```diff
@@ -96,7 +97,8 @@ handle_nl_command_error(
 {
     int *ret = arg;
     *ret = err->error;
```

Comment:
Any chance that this err->error could be == 1 i.e. would not cause exit from loop at suspend_plugin_netlink_handler()?

Reply:
err->errno is a negative errno in case of an error, otherwise 0